wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -46,15 +47,26 @@ def _check_base_class(self, node: ast.ClassDef) -> None:
         if len(node.bases) == 0:
             self.add_violation(RequiredBaseClassViolation(node, text=node.name))
 
+    def _check_extra_object(self, node: ast.ClassDef) -> None:
+        """Check 'object' class in parent list."""
+        if len(node.bases) >= 2:
+            for base_name in node.bases:
+                id_attr = getattr(base_name, 'id', None)
+                if id_attr is not None and id_attr == 'object':
```

Comment:
This line contains a duplication: `id_attr is not None and id_attr == 'object'`. 
When `id_attr == 'object'` it is automatically `is not None`.

Reply:
thank you, i fix it